modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -1349,44 +1351,99 @@ private MediaPackage mergeScheduledMediaPackage(MediaPackage mp) throws IngestEx
+        // The sset managed media package may have a publication element for a live event, if retract live has not run yet.
+        // Publications do not have flavors and are never part of the mediapackage from the capture agent.
+        // Therefore, ignore publication element because it is removed when the recorded media is published and causes complications (on short media) if added.
+        logger.debug("Ignoring {}, not adding to ingested mediapackage {}", MediaPackageElement.Type.Publication, mp);
         continue;
+      } else if (mp.getElementsByFlavor(element.getFlavor()).length > 0) {
+        // The default is to overwrite matching flavored elements in the Asset managed mediapackage (e.g. catalogs)
+        // If isOverwrite is true, changes made from the CA overwrite (update/revert) changes made from the Admin UI.
+        // If isOverwrite is false, changes made from the CA do not overwrite (update/revert) changes made from the Admin UI.
+        // regardless of overwrite, always keep new ingested tracks.
+        if (isOverwrite || MediaPackageElement.Type.Track.equals(element.getElementType())) {
+          // Allow updates made from the Capture Agent to overwrite existing metadata in Opencast
+          logger.debug(
+                  "Omitting existing (Asset Managed) element '{}', replacing with ingested element of same flavor '{}'",
+                  element,
+                  element.getFlavor());
+          continue;
+        }
+        // Remove flavored element from ingested mp and replaced it with maching element from Asset Managed mediapackage.
+        // This protects updates made from the admin UI during an event capture from being reverted by artifacts from the ingested CA.
+        for (MediaPackageElement el : mp.getElementsByFlavor(element.getFlavor())) {
+          logger.info("Omitting ingested element '{}' {}, keeping existing (Asset Managed) element of same flavor '{}'", el, el.getURI(),
+                  element.getFlavor());
+          mp.remove(el);
+        }
       }
-      logger.info("Adding new scheduled element '{}' to ingested mediapackage", element);
+      logger.info("Adding element {} from scheduled (Asset Managed) event '{}' into ingested mediapackage", element, mp);
       mp.add(element);
     }
   }
 
+  /**
+   *
+   * The previous OC behaviour is for metadata in the ingested mediapackage to be updated by the
+   * Asset Managed metadata *only* when the field is blank on the ingested mediapackage.
+   * However, that field may have been intentionally emptied by
+   * removing its value from the Capture Agent UI (e.g. Galicaster)
+   *
+   * If isOverwrite is true, metadata values in the ingest mediapackage take precedence and are *not* replaced Asset Managed metadata fields.
+   * If isOverwrite is false, the ingest mediapackage metadata is updated with Asset Managed mediapackage metadata (i.e. Asset Manged mp data prevails, and can empty out values)
```

Comment:
`and can empty out value`.  Can't both cases lead to that?  If I empty out a field on the CA, and isOverwrite is true, then that field *should* be empty at the end of the day.  Likewise, if I populate a field on the CA, and isOverwrite is false, then that fields should also end up empty.  I'm fine with the logic here, but the comment clouds things.

Reply:
Thanks for that observation, @gregorydlogan , I'll fix the comment.

Reply:
@karendolan do you have time to fix the comment and resolve the merge conflicts today?

Reply:
@JamesUoM I have 30 min right now...